pkg/vulnsrc/amazon/amazon_test.go

```diff
@@ -5,7 +5,6 @@ import (
 	"path/filepath"
 	"testing"
 
-	"github.com/aquasecurity/trivy-db/pkg/db"
```

Comment:
We use this import:
https://github.com/aquasecurity/trivy-db/blob/a897efc986f288eeed502d396dae58692de2db33/pkg/vulnsrc/amazon/amazon_test.go#L155